src/main/java/com/google/graphgeckos/dashboard/components/BuildBotData.java

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.components;
```

Comment:
Using `components` sub-package is confusing for these POJOs: we're using Spring, which has Components. These POJOs are not Spring Components. I'd a `components` sub-package to contain such Spring components.

Reply:
Renamed it